src/prometheus_alert_rules/high_restart_rate.rule

```diff
@@ -0,0 +1,8 @@
+alert: HydraHighRestartRate
+expr: rate(process_start_time_seconds[3m]) > 0.1
```

Comment:
which magic adds the labels so that this query (even though the metric is hydra specific here) doesn't trigger on other applications?as i see that `%%juju_topology%%` is missing

Reply:
For prometheus rules, the labels are automatically filled in to the template. The %%juju_topology%% is something you should only see on loki rules.